### PR TITLE
Switch to building off cimg/node and pre-install meteor

### PR DESCRIPTION
**Pull request by rjmackay**
- cimg/node already contains dockerize and other dependencies
- avoid waiting for meteor install by pre-installing the right version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ https://hub.docker.com/repository/docker/veho/ci-meteor
 
 Build a tagged image
 ```
-docker build -t veho/ci-meteor:1.10 .
+docker build ./ -t veho/ci-meteor:1.10 .
```
Comment (joshhanna) [inline README.md:8]: maybe i'm missing something, but i don't think you need to specify the context multiple times.   does the `./` do something i'm not seeing that the `.` at the end doesn't also do?

Comment (rjmackay) [reply README.md:8]: Ooops. I think I missed that when copy/pasting this command and then though it was broken 🤦 
I'll revert

Comment (rjmackay) [reply README.md]: Fixed

**Comment by rjmackay** [comment]
This is live in dockerhub so going to go ahead an merge it

**Pull request merged**